src/aufgabe1.java

```diff
@@ -0,0 +1,35 @@
+import java.lang.reflect.Array;
+import java.util.Scanner;
+public class aufgabe1 {
+
+   public static void main(String[] args) {
+
+      aufgabe_1();
+      aufgabe_2();
+   }
+
+   private static void aufgabe_1(){
+      Scanner scan = new Scanner(System.in);
+      System.out.print("Write a double number: ");
+      double x = scan.nextDouble();
+
+      for (int i = 1; i < 9; i++) {
```

Comment:
Bonus points for dynamic input! You could use x instead of 9 to always get the complete list. You could use Scanner::nextInt() to receive an integer.